tests/contrib/openai/test_openai_llmobs.py

```diff
@@ -682,6 +682,55 @@ def test_deepseek_as_provider(self, openai, mock_llmobs_writer, mock_tracer):
         assert span_event["meta"]["model_provider"] == "deepseek"
         assert span_event["meta"]["model_name"] == "deepseek-chat"
 
+    @mock.patch("openai._base_client.SyncAPIClient.post")
```

Comment:
This test file is specifically for LLMObs events, not APM spans. Let's make tests in `test_openai.py` instead for asserting APM spans 😄